src/server/status/index.js

```diff
@@ -29,7 +34,8 @@ export default function (kbnServer, server, config) {
           build_snapshot: matchSnapshot.test(config.get('pkg.version'))
         },
         status: kbnServer.status.toJSON(),
-        metrics: kbnServer.metrics
+        metrics: kbnServer.metrics,
+        stats,
```

Comment:
I think @jbudz makes a good point.  Nothing in the status page requires those saved object stats, so we shouldn't be making those extra queries if they aren't needed.  This makes me lean towards a separate `api/stats`.  Thoughts from the monitoring team?